### ISSUE TITLE
Cannot send application logs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [paleg/eventlog-to-syslog#35]
```
What steps will reproduce the problem?
1. A Windows 2008 server has print service started
2.
3.

What is the expected output? What do you see instead?
The Admin/Operational logs under "Application and Services 
Log"->"Microsoft"->"Windows"->"PrintService" didn't send to the syslog server.

What version of the product are you using? On what operating system?
Windows 2008 R2 64bit. Syslog server: Centos 5.5

Please provide any additional information below.
Many thanks for your time!

```

Original issue reported on code.google.com by `jacky11...@gmail.com` on 15 Apr 2011 at 7:25

**Comment by GoogleCodeExporter** [comment]
```
Hmm I will look into this, I have not seen this issue before. I will let you 
know what I find out.
```

Original comment by `sherwin....@gmail.com` on 20 May 2011 at 2:43

**Comment by GoogleCodeExporter** [comment]
```
Did you ever find a resolution to this issue?
```

Original comment by `sherwin....@gmail.com` on 8 Jul 2011 at 3:51

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v4.5.0
```

Original comment by `sherwin....@gmail.com` on 30 Sep 2013 at 4:43
- Changed state: **Fixed**
